programs/intent-transfer/src/be.rs

```diff
@@ -0,0 +1,74 @@
+use anchor_lang::{AnchorDeserialize, AnchorSerialize};
+
+#[derive(PartialEq, Debug)]
+pub struct U16BE(pub u16);
+
+impl AnchorDeserialize for U16BE {
```

Comment:
imo this module should go inside `bridge/` since it's only used by the processor there